roxie/ccd/ccdquery.cpp

```diff
@@ -1917,7 +1934,9 @@ extern IQueryFactory *createServerQueryFactoryFromWu(IConstWorkUnit *wu, const I
         dll.setown(createWuQueryDll(wu));
     if (!dll)
         return NULL;
-    return createServerQueryFactory(wu->queryWuid(), dll.getClear(), queryRootRoxiePackage(), NULL, true, false); // MORE - if use a constant for id might cache better?
+    IQueryFactory *ret = createServerQueryFactory(wu->queryWuid(), dll.getClear(), queryRootRoxiePackage(), NULL, true, false); // MORE - if use a constant for id might cache better?
```

Comment:
Needs to be an Owned<> otherwise an exception thrown from preloadOnce() will cause it to leak

Reply:
Will fix